client/src/views/users/UpdateProfileModal.vue

```diff
@@ -0,0 +1,161 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="modal">
+    <ms-modal
+      title="UsersPage.updateProfile.title"
+      :close-button="{ visible: true }"
+      :confirm-button="{
+        disabled: selectedProfile === undefined,
+        label: 'UsersPage.updateProfile.actionUpdate',
+        onClick: updateProfile,
+      }"
+      :cancel-button="{
+        label: 'UsersPage.updateProfile.actionCancel',
+        disabled: false,
+        onClick: dismissModal,
+      }"
+    >
+      <div
+        class="update-profile-content"
+        v-show="affectedUsers.length > 0"
+      >
+        <div class="update-profile">
+          <ion-text class="update-profile-title body">
+            {{ $msTranslate({ key: 'UsersPage.updateProfile.affectedUsers', count: affectedUsers.length }) }}
+          </ion-text>
+          <div class="update-profile-user">
+            <span
+              v-for="user in affectedUsers.slice(0, 3)"
+              :key="user.id"
+              class="update-profile-user__item subtitles-sm"
+            >
+              {{ user.humanHandle.label }}
+            </span>
+            <span
+              v-show="affectedUsers.length > 3"
+              class="update-profile-user__more body"
+            >
+              {{
+                $msTranslate({
+                  key: 'UsersPage.updateProfile.additionalUsers',
+                  count: affectedUsers.length - 3,
+                  data: { count: affectedUsers.length - 3 },
+                })
+              }}
+            </span>
+          </div>
+        </div>
+        <div
+          class="warn-outsiders"
+          v-show="users.length !== affectedUsers.length"
+        >
+          <ms-report-text :theme="MsReportTheme.Warning">
+            {{ $msTranslate('UsersPage.updateProfile.cannotUpdateOutsiders') }}
```

Comment:
This text shows when an outsider is part of the selection, but their profile will still be modified.

Reply:
Yep good catch, forgot to exclude them.